kusto/ingest/internal/conn/conn.go

```diff
@@ -44,7 +44,7 @@ type Conn struct {
 }
 
 // New returns a new Conn object.
-func New(endpoint string, auth kusto.Authorization) (*Conn, error) {
+func New(endpoint string, auth kusto.Authorization, client *http.Client) (*Conn, error) {
```

Comment:
Should this be called out as a breaking change? Technically, the constructor for the connection changed, right?

Reply:
It's an internal file, so it's not breaking to change it.